enterprise/cypher/acceptance-spec-suite/src/test/scala/org/neo4j/internal/cypher/acceptance/ShortestPathAcceptanceTest.scala

```diff
@@ -743,6 +743,18 @@ class ShortestPathAcceptanceTest extends ExecutionEngineFunSuite with CypherComp
     result.toList should equal(List(Map("length(path)" -> 2)))
   }
 
+  test("should not require named nodes in shortest path") {
+    graph.execute("CREATE (a:Person)-[:WATCH]->(b:Movie)")
+
+    val query =
+      """MATCH p=shortestPath( (:Person)-[*1..4]->(:Movie) )
+        |RETURN length(p)
+      """.stripMargin
+    val result = graph.execute(query)
+
+    println(result.next())
```

Comment:
Same as the other PR, you have forgotten a debug line here and I think it would be nice with an assertion of the result